### PR TITLE
Update the proxy service

### PR DESCRIPTION
**Pull request by mmattel**
References:
https://github.com/owncloud/ocis/pull/5878 ([docs-only] proxy: update README with notes about role assignment)

Updates the proxy service with recent changes from the ocis repo.

Language review welcomed.

### CODE REVIEW

**Review by phil-davis — COMMENTED** [review]
The text LGTM. Someone else can confirm that the detailed content is accurate.

File: modules/ROOT/pages/deployment/services/s-list/proxy.adoc [hunk 1]
```diff
@@ -32,6 +32,54 @@ role_quotas:
...
+            admin: myAdminRole
+            user: myUserRole
+            spaceadmin: mySpaceAdminRole
+            guest: myGuestRole
+```
+
+This would assign the role `admin` to users with the value `myAdminRole` in the claim `ocisRoles`.
+The role `user` to users with the values `myUserRole` in the claim `ocisRoles` and so on.
+
+Claim values that are not mapped to a specific ownCloud Infinite Scale role will be ignored.
```
Comment (EParzefall) [inline modules/ROOT/pages/deployment/services/s-list/proxy.adoc:68]:
```suggestion
Claim values that are not mapped to a specific Infinite Scale role will be ignored.
```

File: modules/ROOT/pages/deployment/services/s-list/proxy.adoc [hunk 2]
```diff
@@ -32,6 +32,54 @@ role_quotas:
...
+
+Note: An ownCloud Infinite Scale user can only have a single role assigned. If the configured
```
Comment (EParzefall) [inline modules/ROOT/pages/deployment/services/s-list/proxy.adoc:70]:
```suggestion
Note: An Infinite Scale user can only have a single role assigned. If the configured
```

File: modules/ROOT/pages/deployment/services/s-list/proxy.adoc [hunk 3]
```diff
@@ -32,6 +32,54 @@ role_quotas:
...
+`role_mapping` and a user's claim values result in multiple possible roles for a user, an error
+will be logged and the user will not be able to login.
```
Comment (EParzefall) [inline modules/ROOT/pages/deployment/services/s-list/proxy.adoc:72]:
```suggestion
will be logged and the user will not be able to log in.
```

**Review by EParzefall — APPROVED** [review]

**Pull request merged**